app/controllers/concerns/pageable_feed_concern.rb

```diff
@@ -0,0 +1,14 @@
+module PageableFeedConcern
+  extend ActiveSupport::Concern
+  include PageableConcern
+
+  private
+
+  def initialize_pageable_feed
+    page = params[:page]&.to_i || 1
+
+    feed_items = Feed.for_page(:title, page, items_per_page)
+    feed_items = Kaminari.paginate_array(feed_items, total_count: Feed.count).page(page).per(items_per_page)
+    @feed = FeedPresenter.new(feed_items: feed_items, current_user: current_user)
+  end
+end
```

Comment:
A simple concern that allows us to create a presenter (really more of a decorator in this case) to be consumed in our views; the feeds made available by this concern are NON-recommended (e.g. NOT-starred); why concern? So we can move it around as we wish: dashboards, widgets, etc.